taqueria-plugin-flextesa/README.md

```diff
@@ -24,8 +41,133 @@ To install the Flextesa plugin on a Taqueria project, navigate to the project fo
 taq install @taqueria/plugin-flextesa
 ```
 
+Once installed, you can confirm functionality by runing the command `taq start sandbox` which will start a sandbox with the default `local` configuration
+
+:::note
+The project you install it on must be initialized by NPM. This only needs to be done once on a project and is accomplished by running the command:
+```shell
+npm init -y
+``` 
+:::
+
+## CLI Commands
+
+The Flextesa plugin allows you to start, stop, and query the sandboxes configured in `./.taq/config.json` from the CLI
+
+The following commands are available from the CLI or the VS Code Command palette:
+- `taq start sandbox [sandboxName]`
+- `taq stop sandbox [sandboxName]`
+- `taq list accounts [sandboxName]`
+
+:::note
+The first time you start a sandbox, it might take several minutes to start. This is normal when starting a Flextesa image
```

Comment:
Perhaps we should explain a little further:
```
The first time you start a sandbox, it might take several minutes to start. This is expected behaviour as it takes time to download the flextesa docker image to your computer.
```
```